While optimizing for pentiumII, __i686 gets defined, resulting
in a compile error "junk at end of line"...
	-- jsaw

--- dietlibc-0.27/i386/PIC.h.orig	2004-12-12 00:41:23.957319464 +0100
+++ dietlibc-0.27/i386/PIC.h	2004-12-12 01:04:36.086683632 +0100
@@ -2,6 +2,8 @@
 #define __I386_PIC_H__
 #ifdef __DYN_LIB
 
+#undef __i686
+
 #if 1
 /* don't trash the athlon return stack */
 #if (__GNUC__ >= 3)	// FIXME: how do I check for the binutils version ?
